generators/php/sdk/src/endpoint/request/WrappedEndpointRequest.ts

```diff
@@ -214,19 +214,46 @@ export class WrappedEndpointRequest extends EndpointRequest {
         paramRef: string;
         property: FileProperty;
     }): void {
-        writer.writeNodeStatement(
-            php.invokeMethod({
-                method: "addPart",
-                arguments_: [
-                    php.invokeMethod({
-                        method: "toMultipartFormDataPart",
-                        arguments_: [php.codeblock(`'${property.key.wireValue}'`)],
-                        on: php.codeblock(paramRef)
-                    })
-                ],
-                on: this.getRequestBodyArgument()
-            })
-        );
+        if (property.contentType != null) {
+            writer.writeNodeStatement(
+                php.invokeMethod({
+                    method: "addPart",
+                    arguments_: [
+                        php.invokeMethod({
+                            method: "toMultipartFormDataPart",
+                            arguments_: [
+                                {
+                                    name: "name",
+                                    assignment: php.codeblock(`'${property.key.wireValue}'`)
+                                },
+                                {
+                                    name: "contentType",
+                                    assignment: php.codeblock(`'${property.contentType}'`)
+                                }
+                            ],
+                            on: php.codeblock(paramRef),
+                            multiline: true
+                        })
+                    ],
+                    on: this.getRequestBodyArgument(),
+                    multiline: true
+                })
+            );
+        } else {
+            writer.writeNodeStatement(
+                php.invokeMethod({
+                    method: "addPart",
+                    arguments_: [
+                        php.invokeMethod({
+                            method: "toMultipartFormDataPart",
+                            arguments_: [php.codeblock(`'${property.key.wireValue}'`)],
+                            on: php.codeblock(paramRef)
+                        })
+                    ],
+                    on: this.getRequestBodyArgument()
+                })
+            );
+        }
```

Comment:
I think we can split this out a bit - the only thing that is changing is the `arguments_` list, so how about this instead?

```typescript
const arguments_ [
  {
    name: "name",
    assignment: php.codeblock(`'${property.key.wireValue}'`)
  }
]
if (property.contentType != null) {
  arguments_.push(
    {
      name: "contentType",
      assignment: php.codeblock(`'${property.contentType}'`)
    }
  )
}
writer.writeNodeStatement(
  ...
)
```

I personally think it's fine to omit `multiline` in the `toMultipartFormDataPart` in both cases (since it's at most two parameters), but if you care to keep the formatting as-is, we could just base it on the length of arguments:

```typescript
{
  ...,
  multiline: arguments_.length > 1
}
```

Reply:
done